File: calculator.js
// Basic Calculator in JavaScript

let num1 = parseFloat(prompt("Enter first number:"));
let num2 = parseFloat(prompt("Enter second number:"));
let operator = prompt("Enter operator (+, -, *, /):");

let result;

if (isNaN(num1) || isNaN(num2)) {
    alert("Please enter valid numbers");
} else {
    switch (operator) {
        case "+":
            result = num1 + num2;
            break;
        case "-":
            result = num1 - num2;
            break;
        case "*":
            result = num1 * num2;
            break;
        case "/":
            if (num2 !== 0) {
                result = num1 / num2;
            } else {
                alert("Division by zero is not allowed");
                result = null;
            }
            break;
        default:
            alert("Invalid operator");
            result = null;
    }

    if (result !== null) {
        alert("Result: " + result);
        console.log("Result:", result);
    }
}

File: calculator.c
#include <stdio.h>

int main() {
    double num1, num2, result;
    char operator;

    printf("Enter first number: ");
    scanf("%lf", &num1);

    printf("Enter second number: ");
    scanf("%lf", &num2);

    printf("Enter operator (+, -, *, /): ");
    scanf(" %c", &operator);   // space before %c handles newline

    switch (operator) {
        case '+':
            result = num1 + num2;
            printf("Result = %.2lf", result);
            break;

        case '-':
            result = num1 - num2;
            printf("Result = %.2lf", result);
            break;

        case '*':
            result = num1 * num2;
            printf("Result = %.2lf", result);
